README.md

```diff
@@ -125,7 +125,7 @@ All other components and packages should be considered development components on
 ### Development branch
 
 The primary development branch is [`develop`](https://github.com/ethereum-optimism/optimism/tree/develop/).
-`develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://community.optimism.io/docs/useful-tools/networks/).
+`develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://docs.optimism.io/builders/node-operators/rollup-node#build-your-node).
```

Comment:
Correct link is here: https://docs.optimism.io/chain/networks

Reply:
Thanks!